iree/turbine/kernel/wave/utils.py

```diff
@@ -912,6 +924,10 @@ def device_randn(*args, **kwargs):
     return torch.randn(*args, **kwargs).to("cuda")
 
 
+def device_randint(*args, **kwargs):
+    return torch.randint(*args, **kwargs).to("cuda")
```

Comment:
As we are handling `torch.cuda.is_available() == False` in `get_default_arch()`, probably should also handle it here and use cpu tensors.

Reply:
done!